packages/owl-bot/.nycrc

```diff
@@ -2,6 +2,7 @@
   "report-dir": "coverage",
   "reporter": ["text", "lcov"],
   "exclude": [
+    "**/src/bin/**",
```

Comment:
our CLI is being used for local testing purposes, let's not count it towards missed coverage.